cmd/mimir/help-all.txt.tmpl

```diff
@@ -1145,6 +1145,12 @@ Usage of ./cmd/mimir/mimir:
     	Per-tenant allowed push request burst size. 0 to disable.
   -distributor.request-rate-limit float
     	Per-tenant push request rate limit in requests per second. 0 to disable.
+  -distributor.retry-after-header.base-seconds int
+    	[experimental] Base duration in seconds for calculating the Retry-After header in responses to 429/5xx errors. (default 3)
+  -distributor.retry-after-header.enabled
+    	[experimental] Enabled controls inclusion of the Retry-After header in the response: true includes it for client retry guidance, false omits it.
+  -distributor.retry-after-header.max-backoff-exponent int
```

Comment:
Is there any reason why the max backoff can't be a simple duration or seconds int?

Reply:
Dimitar and me also pointed this out, there was a discussion in https://github.com/grafana/mimir/pull/6608#discussion_r1389466608

To keep code simple, we decided to go with it as-is for now.

Reply:
I don't think requiring users to do math to determine the max backoff (ex: `base-seconds * 2^max-backoff-exponent`) is simpler than just stating the max backoff. Is there a benefit to requiring a formula to compute the max backoff?